src/main/java/app/library/management/core/domain/util/BookStatusManager.java

```diff
@@ -0,0 +1,27 @@
+package app.library.management.core.domain.util;
+
+import app.library.management.core.domain.Book;
+import app.library.management.core.repository.BookRepository;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class BookStatusManager {
+
+    private static final int POOL_SIZE = 256;
+    ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(POOL_SIZE);
```

Comment:
```suggestion
    private static final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(POOL_SIZE);
```

의도한 바가 하나의 스레드 풀을 생성하고 execute를 호출할 때마다 Thread가 생성된다는 개념으로 설계했다면 static으로 선언해야 하지 않을까 싶습니다.

Reply:
넵! 다음과 같이 변경하였습니다 [a2d21c5](https://github.com/prgrms-be-devcourse/java-library-management/pull/26/commits/a2d21c5bb27dffe8f422bff859b4aa7a1d6f47e4)